e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -178,12 +178,48 @@ describe("scenarios > collection defaults", () => {
     });
   });
 
-  describe("render last edited by when names are null", () => {
-    beforeEach(() => {
-      restore();
-      cy.signInAsAdmin();
-    });
+  describe("description", () => {
+    it('should display collection description', () => {
+      visitRootCollection();
+    
+      cy.get('table').within(() => {
+        cy.findByText("First collection").parent().parent().within(() => {
+          cy.icon("info").trigger("mouseenter"); 
+        })
+      })
+    
+      popover().within(() => {
+        cy.findByText(/Collection First/);
+      })
+    })
 
+    it('should support markdown in collection description', () => {
+      getCollectionIdFromSlug("first_collection", id => {
+        visitCollection(id);
+      });
+
+      cy
+        .findByPlaceholderText("Add description")
+        .clear()
+        .type("**important text** and [link](https://metabase.com)")
+        .blur();
+
+      visitRootCollection();
+
+      cy.get('table').within(() => {
+        cy.findByText("First collection").parent().parent().within(() => {
+          cy.icon("info").trigger("mouseenter"); 
+        })
+      })
+    
+      popover().within(() => {
+        cy.findByText(/important text/);
+        cy.findByText(/link/);
+      })
```

Comment:
These assertions would pass if we revert the `<Markdown />` component inside the `BaseTableItem`. I'm not sure there's a good way to guarantee formatting is correct, but maybe we could also assert there are no asterisks or brackets? Hopefully that'd give us a bit more confidence here